hbp_knowledge/receptors/zoli2018.bel

```diff
@@ -83,214 +92,337 @@ a(MESH:Acetylcholine) -- bp(MESH:Apoptosis)
+SET Confidence = "Medium"
 a(MESH:Erythrocytes) -> a(MESH:Acetylcholine)
 a(MESH:"Endothelial Cells") -> a(MESH:Acetylcholine)
 a(MESH:"Epithelial Cells") -> a(MESH:Acetylcholine)
+a(MESH:Placenta) -> a(MESH:Acetylcholine)
+a(MESH:Placenta) -> a(MESH:"Choline O-Acetyltransferase")
+#couldn't find placental cells in MESH. Hence, added placenta.
 a(MESH:Erythrocytes) -> a(MESH:"Choline O-Acetyltransferase")
 a(MESH:"Endothelial Cells") -> a(MESH:"Choline O-Acetyltransferase")
 a(MESH:"Epithelial Cells") -> a(MESH:"Choline O-Acetyltransferase")
+UNSET Confidence
 
 SET Evidence ="In the brain, nAChRs are widely expressed, both presyn- aptically and postsynaptically, and are involved in several functions including learning and memory, arousal, reward, motor control, and analgesia. nAChRs are also the target of nicotine, the main addictive agent delivered by cigarette smoke"
+SET Confidence = "Medium"
 p(MESH:"Receptors, Nicotinic") -- path(MESH:Learning)
 p(MESH:"Receptors, Nicotinic") -- path(MESH:Memory)
 p(MESH:"Receptors, Nicotinic") -- path(MESH:Arousal)
 p(MESH:"Receptors, Nicotinic")-- path(MESH:Reward)
+p(MESH:"Receptors, Nicotinic")-- path(MESH:"Motor Activity")
+#instead of motor control, motor activity was used
 p(MESH:"Receptors, Nicotinic") -- bp(MESH:Analgesia)
+UNSET Confidence
 
 SET Evidence ="Homomeric α7 receptors are one of the two most abun- dant nAChR subtypes in the CNS, where they are mainly localised in the cortex, hippocampus, hypothalamus and some brain stem nuclei."
+SET Confidence = "High"
 p(HGNC:CHRNA7,loc(MESH:"Cerebral Cortex"))
 p(HGNC:CHRNA7,loc(MESH:Hippocampus))
 p(HGNC:CHRNA7,loc(MESH:Hypothalamus))
+UNSET Confidence
 
 SET Evidence ="In heterologous sys- tems, α7 subunits form functional channels with the β2 [29], β3 [30], β4 [31], or α5 subunit [32]. "
-complex(p(HGNC:CHRNA7),p(HGNC:CHRNB2),p(HGNC:CHRNB3),p(HGNC:CHRNB4),p(HGNC:CHRNA5))
+SET Confidence = "Medium"
+composite(p(HGNC:CHRNA7),p(HGNC:CHRNB2)) -> complex(p(HGNC:CHRNA7),p(HGNC:CHRNB2))
+composite(p(HGNC:CHRNA7),p(HGNC:CHRNB3)) -> complex(p(HGNC:CHRNA7),p(HGNC:CHRNB3))
+composite(p(HGNC:CHRNA7),p(HGNC:CHRNB4)) -> complex(p(HGNC:CHRNA7),p(HGNC:CHRNB4))
+composite(p(HGNC:CHRNA7),p(HGNC:CHRNA5)) -> complex(p(HGNC:CHRNA7),p(HGNC:CHRNA5))
+#the evidence says alpha7 combination with rest subunits. The following line in the paper says "our group has recently biochemically identified a native α7β2 subtype expressed in rodent and human basal forebrain that has different functional and pharmacological properties from those of homomeric α7 nAChRs". Hence, I assumed that rest subunits are in individual combination with alpha7.
+UNSET Confidence
 
 SET Evidence ="CHRFAM7A mRNA expression is low in human brain, but abundant in peripheral lymphocytes and tissues"
-r(HGNC:CHRFAM7A,loc(MESH:"Leukemia, Large Granular Lymphocytic"))
+SET Confidence = "Medium"
+SET Anatomy = "peripheral nervous system"
+r(HGNC:CHRFAM7A,loc(MESH:Lymphocytes))
```

Comment:
maybe I would go with association, because in this case you would get a relationship between the RNA and it's place of expression...
```suggestion
r(HGNC:CHRFAM7A) -- a(MESH:Lymphocytes)
```